apiclient/harvester_api/managers.py

```diff
@@ -203,6 +203,13 @@ def delete(self, name, namespace=DEFAULT_NAMESPACE, *, raw=False):
         path = self.PATH_fmt.format(uid=f"/{name}", ns=namespace)
         return self._delete(path, raw=raw)
 
+    def export(self, volume_name, image_name, namespace=DEFAULT_NAMESPACE, *, raw=False):
+        export_spec = {"displayName": image_name,
+                       "namespace": namespace, "storageClassName": "harvester-longhorn"}
+
+        path = self.PATH_fmt.format(uid="", ns=namespace) + "/{}?action=export".format(volume_name)
```

Comment:
We should use `params` for query string instead of string concatenation.
```python
path = self.PATH_fmt.format(uid="", ns=namespace) + "/{}?action=export".format(volume_name)
# equals to
path = self.PATH_fmt.format(uid=f"/{volume_name}", ns=namespace)
# without `?action=export` query string
```

`VirtualMachineManager` have some examples for query string, it would be something like
```python
params = dict(action="export")
self._create(path, params=params)
```

Reply:
Update related lines according to comment